backend/app/main/views.py

```diff
@@ -8,3 +10,23 @@
 @main.route("/", methods=["GET"])
 def index():
     return "Hello World!"
+
+
+# get all mentors from Airtable
+@main.route("/mentors", methods=["GET"])
+def get_all_mentors():
```

Comment:
Airtable does a thing where they only return 10 records at a time with each request, so we actually need to make multiple requests if we have more than 10 rows. We can do this later though